### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by hpryce**
Updates readme to account for the new features that have been added since the last release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+    2. Drag and drop the “log4j-sniffer” icon into Downloads.
+ ![dragging and dropping](https://github.com/palantir/log4j-sniffer/blob/develop/mac_download.png?raw=true)
+
+##### Use log4j-sniffer
+
+1. Open the Terminal by searching for “Terminal” using the magnifying glass in the top right corner of the screen.
+2. Run `~/Downloads/log4j-sniffer crawl ~/ --ignore-dir="^/dev"` to crawl the entire system
+    1. Run `~/Downloads/log4j-sniffer crawl ~/PATH/TO/YOUR/FOLDER` to crawl specific folders
+    2. If your computer is unable to locate log4j-sniffer, you may have to make it executable before using it. In your terminal, run the following:
+        1. `chmod +x log4j-sniffer `
```
Comment (jasonknudsen) [inline README.md]:
```suggestion
        1. `chmod +x ~/Downloads/log4j-sniffer `
```

File: README.md [hunk 2]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+1. Once the download is complete, click on the file to open.
+1. Drag and drop the “log4j-sniffer” icon into your Downloads through the Finder.
+    1. Open a Finder window by searching for “Finder” using the magnifying glass on the top right of your screen, or selecting the icon in your Dock.
+    2. Drag and drop the “log4j-sniffer” icon into Downloads.
+ ![dragging and dropping](https://github.com/palantir/log4j-sniffer/blob/develop/mac_download.png?raw=true)
+
+##### Use log4j-sniffer
+
+1. Open the Terminal by searching for “Terminal” using the magnifying glass in the top right corner of the screen.
+2. Run `~/Downloads/log4j-sniffer crawl ~/ --ignore-dir="^/dev"` to crawl the entire system
```
Comment (jasonknudsen) [inline README.md]:
```suggestion
2. Run `~/Downloads/log4j-sniffer crawl / --ignore-dir="^/dev"` to crawl the entire system
```

Comment (jasonknudsen) [reply README.md]: Adding ~ will ensure it only scans the Users folder, think we want to recommend scanning the whole system?

File: README.md [hunk 3]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+    1. Run `~/Downloads/log4j-sniffer crawl ~/PATH/TO/YOUR/FOLDER` to crawl specific folders
```
Comment (jasonknudsen) [inline README.md]:
```suggestion
    1. Run `~/Downloads/log4j-sniffer crawl /PATH/TO/YOUR/FOLDER` to crawl specific folders
```

File: README.md [hunk 4]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+    2. If your computer is unable to locate log4j-sniffer, you may have to make it executable before using it. In your terminal, run the following:
+        1. `chmod +x log4j-sniffer `
+        2. `./log4j-sniffer crawl ~/PATH/TO/YOUR/FOLDER`
```
Comment (jasonknudsen) [inline README.md]:
```suggestion
        2. `./log4j-sniffer crawl /PATH/TO/YOUR/FOLDER`
```

File: README.md [hunk 5]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+##### Download the latest version
+
+1. Locate log4j-sniffer/releases (https://github.com/palantir/log4j-sniffer/releases).
+2. Select the Windows asset.
+3. Confirm that the file is downloading to your “Downloads” folder.
+
+##### Use log4j-sniffer
+
+1. Type "Command Prompt" into the search bar at the bottom and in the right pane click "Run as administrator".
+2. Navigate to your Downloads folder, e.g. `cd C:\Users\user1\Downloads`
```
Comment (jasonknudsen) [inline README.md]:
```suggestion
2. Navigate to your Downloads folder, e.g. `cd C:\Users\yourname\Downloads`
```

File: README.md [hunk 6]
```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
...
+3. Run `log4j-sniffer-0.8.0-windows-amd64.exe crawl C:\` to crawl the entire system
+    1. Run `log4j-sniffer-0.8.0-windows-amd64.exe crawl C:\PATH\TO\YOUR\FOLDER` to crawl specific folders
+
```
Comment (jasonknudsen) [inline README.md:111]: 2. Scan other drives like D:\ ?

**Review by jasonknudsen — APPROVED** [review]

**Review by nkeoleian — APPROVED** [review]

**Review by nmiyake — APPROVED** [review]

**Pull request merged**